probe.go

```diff
@@ -744,6 +745,10 @@ func (p *Probe) getNetlinkSocket() (*NetlinkSocket, error) {
 
 // attachWithKprobeEvents attaches the kprobe using the kprobes_events ABI
 func (p *Probe) attachWithKprobeEvents() error {
+	if p.kprobeEventUnavailable {
+		return fmt.Errorf("couldn't enable kprobe: kprobe_event is not available")
```

Comment:
Actually the error is not that `kprobe_event` is not available, it's that the hook point doesn't exist. Would you mind switching the error message and attribute name ? (+ bonus points if you put the hardcoded error in `err.go`)